app/javascript/locales.js

```diff
@@ -0,0 +1,25 @@
+const messages = {
+  en: {
+    message: {
+      selector: {
+        button: 'Go to dashboard',
+      },
+    },
+  },
+  es: {
+    message: {
+      selector: {
+        button: 'Ir a dashboard',
+      },
+    },
+  },
+  ja: {
+    message: {
+      selector: {
+        button: 'こんにちは、世界',
+      },
+    },
+  },
+};
+
+export default { messages};
```

Comment:
A space is required before '}'  object-curly-spacing

Reply:
te odio mono!